.github/workflows/check-cherry-pick.yml

```diff
@@ -0,0 +1,19 @@
+name: Check Cherry Pick
+
+on:
+  pull_request:
+    types: [synchronize, opened, labeled]
+
+jobs:
+  rebase-pr:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v1
+      - name: Check Cherry Pick
+        if: >
+          contains(github.pull_request.labels.*.name, 'PR target: master & patch') ||
+          contains(github.pull_request.labels.*.name, 'PR target: patch')
+        uses: ./github-actions/check-cherry-pick
+        env:
+          TARGET: master
```

Comment:
Can't we just take the branch from the GitHub event payload (at `$GITHUB_EVENT_PATH`) from `.pull_request.base.ref`?

Reply:
Nevermind, for some reason I though this was about rebasing on the target branch 🤷‍♂